src/tdastro/astro_utils/dustmap.py

```diff
@@ -8,6 +8,7 @@
 import numpy as np
 from astropy.coordinates import SkyCoord
 from citation_compass import CiteClass
+from sfdmap2 import sfdmap
```

Comment:
For the tests to pass, you will need to include `sfdmap2` in the pyprojects.toml dependencies section.